src/courses/templates/courses/chart_mark_distribution.html

```diff
@@ -1,105 +1,105 @@
-<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.5.0/Chart.min.js"></script>
-<!-- https://github.com/chartjs/Chart.Zoom.js/blob/master/samples/zoom.html -->
-<!-- <script type="text/javascript" src="https://raw.githubusercontent.com/chartjs/Chart.Zoom.js/master/Chart.Zoom.min.js"></script></script> -->
-
 <script type="text/javascript">
+  //{% if request.user.pk == user.pk %}
```

Comment:
Why the comments?  It should work with or without, since the template is processed before the JS, just curious.

Reply:
vs code gives me annoying red warning signs when i use django template tags, commenting it out silences it. I can remove it once i commit

Reply:
now using django syntax extension